docs/src/pages/guides/new-notifications.md

```diff
@@ -0,0 +1,173 @@
+
+```typescript
+export const sendGoogleChat = async (
+  data: GoogleChatData,
+  message: NotificationMessage
+): Promise<any> => {
+
+  const notifType =
+    message.meta.type[0].toUpperCase() + message.meta.type.substring(1)
+
+  switch (message.meta.type) {
+    case 'start':
+      ... code to send start notifications
+    case 'termination':
+      ... code to send termination notifications
+    case `incident`:
+      ... code to send incident notifications
+
+```
+
+This is where you can get creative and really customize the look and feel of the notifications. You can use icons, widgets, fonts and colors of your choice. In the example `googlechat.ts` below, you can see how headers, text coloring are used to create the incident notifications. Your choices and available options will vary depending on your application. Please refer to your specific development documentation.
+
+```typescript
+case 'incident':
+      chatMessage = {
+        cards: [
+          {
+            header: {
+              title: 'Monika Notification',
+              subtitle: `New ${notifType} from Monika`,
+              imageUrl: 'https://bit.ly/3kckaGO',
+            },
+            sections: [
+              {
+                widgets: [
+                  {
+                    textParagraph: {
+                      text: `<b>Message: <font color=#ff0000>Alert!</font></b> ${message.summary}`,
+                    },
+                  },
+                  {
+                    textParagraph: {
+                      text: `<b>URL:</b> <a href>${message.meta.url}</a>`,
+                    },
+                  },
+                  {
+                    textParagraph: {
+                      text: `<b>Time:</b> ${message.meta.time}`,
+                    },
+                  },
+                  {
+                    textParagraph: {
+                      text: `<b>From:</b> ${message.meta.monikaInstance}`,
+                    },
+                  },
+                ],
+              },
+            ],
+          },
+        ],
+      }
+      break
+```
+
+## Validating
+
+Finally, head over to `src/components/config/validate.ts` and look for the `validateNotification()` function. We'd like to make sure Monika is configured properly before each run and `validate.ts` handles the checking. This function is called at the beginning to verify that the user's configuration contains no error.
+
+```typescript
+    case 'google-chat': {
+        if (!notification.data.url) return WEBHOOK_NO_URL
+        break
+      }
+```
+
+In the example above, we made sure that the `google-chat` identifier is properly labeled, and that the `url` field is set. Otherwise a standardized "no-url-found" message is displayed.
+
+## Testing
+
+Don't be lazy, add your unit test(s) into the folder `test/components/notification.test.ts` to ensure that all your code behaves as designed. We use `mocha` testing framework and `chai`'s expect for assertion and checking.
```

Comment:
Don't be lazy, -> To make sure your integration won't break in the future, add your unit test(s) into the folder `test/components/notification.test.ts`. This will also ensure that all your code behaves as designed. 

mocha -> link to mocha
chai -> link to chai